pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -100,10 +112,20 @@ type ConnectionSignature struct {
 	Signature  string `json:"signature,omitempty"`
 	SignedData string `json:"sig_data,omitempty"`
 	SignVerKey string `json:"signers,omitempty"`
+	// InteropKey Interop: signer key compatible with aca-py.
+	InteropKey string `json:"signer,omitempty"`
```

Comment:
What's InteropKey?

Reply:
ACA-py _was_ expecting the connection signature's `signer` field to contain the base58-encoded sender verification key.

Just ran a test to verify if it's still necessary (commented this out and ran the interop test) and it looks like this issue is no longer present, so I'll remove the code.

Reply:
removed this